src/main/java/com/google/codeu/servlets/TranslationServlet.java

```diff
@@ -0,0 +1,30 @@
+import com.google.cloud.translate.Translate;
+import com.google.cloud.translate.TranslateOptions;
+import com.google.cloud.translate.Translation;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/translate")
+public class TranslationServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the request parameters.
+    String originalText = request.getParameter("text");
```

Comment:
Is there a possible need to sanitize this text? From what I can see it is received directly from user input but I don't know if the translate API has some sort of sanitize built into it

Reply:
I don't think it will be necessary, but what kind of sanitization are you thinking about?

Reply:
Sanitization of the text input by a user. In the cases that user(s) input html tags along with scripts with the intention to cause unexpected behaviours when the translated text is shown on the page.

Reply:
I don't think the API has it cause it was already in the given code.